components/my-account/AddressBook/AddressBook.styles.tsx

```diff
@@ -0,0 +1,46 @@
+import { Theme } from '@emotion/react'
+
+import theme, { themeBackGround, themeBorder, grey } from '@/styles/theme'
+
+export const AddressBoxStyle = {
+  AddressBox: {
+    padding: '10px 15px 10px 0',
+  },
+  AddressSection: {
```

Comment:
Create global variant for Box and add CSS mentioned below to that. Using same variant where ever you need to use this CSS code
 
    display: 'flex',
    justifyContent: 'space-between',

Replace this code everywhere

Reply:
this code is for the address section only  , i will create another code for Box